js/exporter/jspdf/export.js

```diff
@@ -125,24 +123,23 @@ export const Export = {
         }
     },
 
-    setColumnWidths: function(autoTableOptions, columns, keepColumnWidths) {
-        const columnStyles = autoTableOptions.columnStyles;
-
-        for(let i = 0; i < columns.length; i++) {
-            columnStyles[i] = columnStyles[i] || {};
-
-            const columnWidth = columns[i].gridColumn.width;
-            if(keepColumnWidths && (typeof columnWidth === 'number') && isFinite(columnWidth)) {
-                columnStyles[i].cellWidth = this.convertPixelsToPoints(columnWidth);
-            } else {
-                columnStyles[i].cellWidth = 'auto';
-            }
+    getDefaultPdfColumnWidths(autoTableWidth, columnWidths) {
+        if(isNumeric(autoTableWidth)) {
+            const tableWidth = columnWidths.reduce((a, b) => { return a + b; });
```

Comment:
I suggest that we should to set InitialValue paremeter for reduce function. In case when columnWidths is [] we will get the exception.

Reply:
changed to
```
const tableWidth = columnWidths.reduce((a, b) => a + b, 0);
```